VAMobile/src/screens/HealthScreen/SecureMessaging/CernerAlertSM/CernerAlertSM.test.tsx

```diff
@@ -14,95 +13,78 @@ jest.mock('utils/hooks', () => {
   }
 })
 
-context('CernerAlertSM', () => {
-  const mockSingleFacilityPatientState: PatientState = {
-    isCernerPatient: true,
-    cernerFacilities: [
-      {
-        isCerner: true,
-        facilityId: '1',
-        facilityName: 'FacilityOne',
-      },
-    ],
-    facilities: [
-      {
-        isCerner: true,
-        facilityId: '1',
-        facilityName: 'FacilityOne',
-      },
-      {
-        isCerner: false,
-        facilityId: '2',
-        facilityName: 'FacilityTwo',
-      },
-    ],
-  }
-  const mockMultipleFacilityPatientState: PatientState = {
-    isCernerPatient: true,
-    cernerFacilities: [
-      {
-        isCerner: true,
-        facilityId: '1',
-        facilityName: 'FacilityOne',
-      },
-      {
-        isCerner: true,
-        facilityId: '2',
-        facilityName: 'FacilityTwo',
-      },
-    ],
-    facilities: [
-      {
-        isCerner: true,
-        facilityId: '1',
-        facilityName: 'FacilityOne',
-      },
-      {
-        isCerner: true,
-        facilityId: '2',
-        facilityName: 'FacilityTwo',
-      },
-    ],
+jest.mock('../../../../api/facilities/getFacilitiesInfo', () => {
+  let original = jest.requireActual('../../../../api/facilities/getFacilitiesInfo')
```

Comment:
Can this just be `api/facilities/getFacilitiesInfo` without all the `../`es?

Reply:
no it unfortunately can't I tried that and it wasn't working